assets/js/vue/components/search/DatasetRow.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+    <b-card class="card-product">
+        <b-link class="search-result"
+                :href="url" target="_blank">
+            <div>
+                <span class="badge badge-secondary" v-if="datasetRowData.availabilityStatus === 0">Identified</span>
+                <span class="badge badge-primary" v-else-if="datasetRowData.availabilityStatus === 2 || datasetRowData.availabilityStatus === 4">Submitted</span>
+                <span class="badge badge-danger" v-else-if="datasetRowData.availabilityStatus === 5 || datasetRowData.availabilityStatus === 8">Restricted</span>
+                <span class="badge badge-success" v-else-if="datasetRowData.availabilityStatus === 7 || datasetRowData.availabilityStatus === 10">Available</span>
+            </div>
+            <b-card-title>{{ datasetRowData.title }}</b-card-title>
+            <b-card-text class="d-flex justify-content-between" >
+                <div v-if="Object.keys(datasetRowData.datasetSubmission).length > 0">
+                    <div v-if="datasetRowData.datasetSubmission.authors">
+                        Authors: {{ datasetRowData.datasetSubmission.authors }}
+                    </div>
+                    <div v-if="datasetRowData.acceptedDate">
+                        Published on {{ datasetRowData.acceptedDate }}
+                    </div>
+                    <div v-if="datasetRowData.fileFormat">
+                        File Format: {{ datasetRowData.fileFormat }}
+                    </div>
+                </div>
+                <div>
+                    <div v-if="datasetRowData.doi.doi">
+                        DOI: {{ datasetRowData.doi.doi }}
+                    </div>
+                    <div>
+                        UDI: {{ datasetRowData.udi }}
+                    </div>
+                    <div v-if="datasetRowData.fileSize">
+                        File Size: {{ datasetRowData.fileSize }}
+                    </div>
+                </div>
+            </b-card-text>
+        </b-link>
+    </b-card>
+</template>
+
+<script>
+    export default {
+        name: "DatasetRow",
+        props: {
+            datasetRowData: {
+                type: Object
+            }
+        },
+        data: function () {
+            return {
+                url: Routing.generate("pelagos_app_ui_dataland_default", {'udi' : this.datasetRowData.udi } )
+            }
+        }
+    }
+</script>
+
+<style scoped>
+    div.bootstrap p {
```

Comment:
Where is the P?

Reply:
Old code when this row had citation text, good catch!